.git-blame-ignore-revs

```diff
@@ -144,3 +144,8 @@ ff5c8f6cc3d1f2e017e86d50965c14b71f00567b
 
 # mangal: format with nixfmt-rfc-style #328284
 3bb5e993cac3a6e1c3056d2bc9bf43eb2c7a5951
+
+# ollama: format with nixfmt-rfc-style and minor changes (#329353)
+7586e9c84427b4d5bb2063daf07b553fcc1a4853
+dc45fb5f8a070722a9cd46ebf6c67b21dccffb6f
+d88e82cf68eb021aba5cc91c0356664877693862
```

Comment:
None of these hashes match any of this pr's current commits.

Reply:
Also, maybe it's better to only add the reformat commit here? I would think minor changes should still show up in `git blame`, and that .git-blame-ignore-revs is best used for noisy but syntactically equivalent changes like reformatting entire files.

Reply:
OK, I merged the small changes in the reformat commit so that there's only one commit in `.git-blame-ignore-revs`.

Reply:
I think having a separate reformat commit was better.

Reply:
Let me know if you prefer to split them and I'll do it gladly.

Reply:
Yeah, I think it would be better to have one commit running `nixfmt`, and another that moves the assert and switches to `with` (I honestly would personally prefer each in its own commit, but from what you've said before fewer commits is generally preferred).

Reply:
Let me quickly do that.

Reply:
Let me know if it's OK now.